lms/static/scripts/frontend_apps/components/StudentSelector.js

```diff
@@ -72,15 +74,21 @@ export default function StudentSelector({
         disabled={!hasPrevView}
         onClick={onPrevView}
       >
-        <img src="/static/images/arrow-left.svg" />
+        <SvgIcon
+          src={require('../../../images/arrow-left.svg')}
```

Comment:
The suggestion I mentioned on the call about handling the SVG markup here was to provide a way for `SvgIcon` to verify that the markup comes from a trusted source which is known not to include any inline event/styles etc. Unlike `<img src="...">` the browser will apply them in this context.

It looks like the `stringify` transform doesn't mark the string in any way to indicate that it came from the JS bundle, so I was thinking along the lines of a helper function to tag the markup. For example, something like:

```js

/**
 * Wrap `markup` to indicate that it came from a trusted source
 * (ie. not unsanitized user input)
 *
 * Returns an object with a `trustedHTML` property whose value is `markup`.
 */
function trustMarkup(markup) {
  return { trustedHTML: markup };
}

// in SvgIcon.js ...

function SvgIcon({ src }) {
  const markup = {
    __html: src.trustedHTML,
  };
}

// in a module that uses SvgIcon ...

<SvgIcon
  src={trustMarkup(require('path/to/icon.svg'))}
/>
```